script.js

```diff
@@ -1,85 +1,90 @@
 const featureDetials = [
-    {
-        image: 'images/I1.jpg',
-        name: 'Edureka',
-        post: 'YouTube Channel',
-        description: 'Edureka is online Youtube Channel which has Web Devlopment and many other courses videos'
-    }
-    , {
-        image: 'images/I2.jpg',
-        name: 'Oniline Tutorials',
-        post: 'YouTube Channel',
-        description: 'Oniline Tutorials is online Youtube Channel which has Web Devlopment mainly focus on Css.'
-    }
-    , {
-        image: 'images/I3.webp',
-        name: 'Web Dev Simplified',
-        post: 'YouTube Channel',
-        description: 'Web Dev Simplified  is online Youtube Channel which has Web Devplopment CSS videos.'
-    }
-    , {
-        image: 'images/I4.png',
-        name: 'Udemy',
-        post: 'Online Website',
-        description: 'Udemy is online Website  which has Web Devlopment and many other courses videos with lectures and notes.'
-    }
-    , {
-        image: 'images/I5.png',
-        name: 'Skill Share',
-        post: 'Online Website',
-        description: 'Skill share  is online Website which has Web Devlopment and many other courses videos and lectures.'
-    }
-    , {
-        image: 'images/I6.png',
-        name: 'Coursea',
-        post: 'Online Website',
-        description: 'Coursea is online Website which has Web Devlopment and many other courses videos and lecture with Notes.'
-    }
-]
+  {
+    image: 'images/I1.jpg',
+    name: 'Edureka',
+    post: 'YouTube Channel',
+    description: 'Edureka is online Youtube Channel which has Web Devlopment and many other courses videos',
+  },
+  {
+    image: 'images/I2.jpg',
+    name: 'Oniline Tutorials',
+    post: 'YouTube Channel',
+    description: 'Oniline Tutorials is online Youtube Channel which has Web Devlopment mainly focus on Css.',
+  },
+  {
+    image: 'images/I3.webp',
+    name: 'Web Dev Simplified',
+    post: 'YouTube Channel',
+    description: 'Web Dev Simplified  is online Youtube Channel which has Web Devplopment CSS videos.',
+  },
+  {
+    image: 'images/I4.png',
+    name: 'Udemy',
+    post: 'Online Website',
+    description: 'Udemy is online Website  which has Web Devlopment and many other courses videos with lectures and notes.',
+  },
+  {
+    image: 'images/I5.png',
+    name: 'Skill Share',
+    post: 'Online Website',
+    description: 'Skill share  is online Website which has Web Devlopment and many other courses videos and lectures.',
+  },
+  {
+    image: 'images/I6.png',
+    name: 'Coursea',
+    post: 'Online Website',
+    description: 'Coursea is online Website which has Web Devlopment and many other courses videos and lecture with Notes.',
+  },
+];
 
 console.log(featureDetials);
```

Comment:
- [ ] Kindly, Remove all `console.log()` you have in this file to keep your code clean and follow [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

![image](https://user-images.githubusercontent.com/45231276/181492387-2a752af3-fff6-4b4f-b3e8-c091ce19fab1.png)